hubblestack_nebula_v2/top.mask

```diff
@@ -0,0 +1,3 @@
+nebula:
+  '*':
```

Comment:
I recently changed the nebula get_top_data to expect the matches as a list of dictionaries. This way ordering is guaranteed. I think you need to change this line as a result:

```
  - '*':
```

Reply:
Done